### PR TITLE
Largely removes chem crates from the trade system.

### PR DESCRIPTION
**Pull request by dustylens**
## About the PR

Removes chem crate restocks from the trade station as per directive from people who hide in the shadows.

Maintains a restock crate of chemicals that either lack a proper source or can only be sourced in trace amounts by current methods. Re-examine when those sources are more commonly available.

## Why / Balance

Part of the ongoing movement towards segregating medical resources towards a more centralized source and to remove some mechanical incentives towards juggistry.

## How to test

Buy crate. Gaze in awe. Ask yourself what we're even spending our spesos on anymore. Remember better times when we were young.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

🆑 
- remove: majority of chemistry jugs removed from sale, to be replaced with new barrels.

### CODE REVIEW

**Comment by dvir001** [comment]
Just ```abstract: true # Frontier``` like others on this file

**Comment by Houtblokje** [comment]
I am sorry, what? What is trade even for at this point, if all you can get there is paperwork? Also, there really aren't good sources of chemicals right now, unless you want to push medical ships even more toward salvaging? Or they would have to buy them from other players, probably for very high prices, if they are even able to get in contact with players who have the chemicals.

File: Resources/Prototypes/_NF/Catalog/Cargo/cargo_medical.yml [hunk 1]
```diff
@@ -7,3 +7,13 @@
...
   category: cargoproduct-category-name-medical
   group: market
+
+- type: cargoProduct
+  id: ChemistryDL
+  icon:
+    sprite: Structures/Storage/Crates/chemcrate_secure.rsi
+    state: icon
+  product: CrateChemistryDL
+  cost: 750
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Cargo/cargo_medical.yml:17]: Might honestly make this more expensive - if a base jug had a value of 750 (per welding fuel), and if you only wanted maybe half of the things in here, maybe 2500 or so?  Somewhere between 3 and 4?

File: Resources/Prototypes/_NF/Catalog/Fills/Crates/chemistry.yml [hunk 2]
```diff
@@ -8,3 +8,28 @@
       contents:
         - id: JugSpaceCleaner
           amount: 5
+
+- type: entity
+  id: CrateChemistryDL
+  parent: CrateChemistrySecure
+  name: chemicals crate (DL)
+  description: Contains chemicals from the bargain bin overstock-block of elements. Requires understanding of hinges to open.
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Crates/chemistry.yml:16]: DL... hmmm... hmmmmmmm...

Jokes aside, might honestly just make it say something like this.  Not verbatim, but something along these lines.
```suggestion
  name: chemicals crate (uncommon)
  description: Contains chemicals that rarely occur in nature. Requires understanding of hinges to open.
```

**Review by whatston3 — COMMENTED** [review]
Seems alright, left a recommendation on the pricing and description of the crate.

**Comment by chrome-cirrus** [comment]
Ooo... Electrolysis units, centrifuges, and condensers about to get a lot more exercise.

**Comment by dvir001** [comment]
Keep in mind I didnt remove the chem crates yet, so I wont say this PR isnt needed.

**Comment by dvir001** [comment]
Now this PR still valid

**Review by whatston3 — APPROVED** [review]
Thanks, Dusty.

**Pull request merged**